app/app/utils.py

```diff
@@ -176,8 +176,7 @@ def sync_profile(handle, user=None, hide_profile=True):
     handle = handle.strip().replace('@', '')
     data = get_user(handle)
     email = ''
-    is_error = 'name' not in data.keys()
-    if is_error:
+    if 'name' not in data.keys():
```

Comment:
i personally prefer the specific `is_error` bc i think the naming convention tells you something about  the view... but i dont feel super super strongly about it